cmd/src/gateway_benchmark.go

```diff
@@ -0,0 +1,258 @@
+package main
+
+import (
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"io"
+	"net/http"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/sourcegraph/src-cli/internal/cmderrors"
+)
+
+const (
+	defaultRequestCount = 100
+	blue                = "\033[34m"
+	green       = "\033[32m"
+	yellow      = "\033[33m"
+	red         = "\033[31m"
+	reset       = "\033[0m"
+)
+
+func init() {
+	usage := `
+'src gateway benchmark' runs performance benchmarks against Cody Gateway endpoints.
+
+Usage:
+
+    src gateway benchmark [flags]
+
+Examples:
+
+    $ src gateway benchmark
+    $ src gateway benchmark --requests 50
+`
+
+	flagSet := flag.NewFlagSet("benchmark", flag.ExitOnError)
+
+	var (
+		requestCount = flagSet.Int("requests", defaultRequestCount, "Number of requests to make per endpoint")
```

Comment:
Maybe just write `defaultRequestCount` inline here instead of a constant (easier to see what the defautl is that way, less likely to conflict with other constants / files in the folder)

Reply:
Also may want to bump this to something statistically significant (e.x. 1000)

Reply:
Not a fan of inlineing magic values, but we only used it once at the relative top of the file so I guess it's okay. Inlined it and bumped it to 1000.